src/protocol/templates/pingmessage.py.in

```diff
@@ -3,10 +3,10 @@
 # {{class_info.name}}.py
 # Python implementation of the Blue Robotics 'Ping' binary message protocol
 
-#~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+# ~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
 # THIS IS AN AUTOGENERATED FILE
 # DO NOT EDIT
```

Comment:
Same here?

Reply:
yes, this is the template generating the autogenerated file.